content/methods/discover/contextual-inquiry.md

```diff
@@ -1,10 +1,20 @@
 ---
 title: Contextual Inquiry
 permalink: /methods/discover/contextual-inquiry/
+redirect_from:
+  - contextual-inquiry/
 layout: layouts/page
 tags: methods
 eleventyNavigation:
   key: methods
   parent: Discover
   title: Contextual Inquiry
+method:
+  title: Contextual Inquiry
+  what: The product team unobtrusively observes participants at work, with their permission, then asks questions.
+  why: To learn how and why users do what they do; to discover needs and attitudes that might not emerge in an <a href="https://guides.18f.gov/methods/discover/stakeholder-and-user-interviews/" class="usa-link">interview</a> to map how tools, digital and otherwise, interact during complex activities.
```

Comment:
[comment] There's some inconsistency around `class="usa-link"`, which appears here but which was removed from the contextual inquiry method.  Was this intentional?

Reply:
Good catch. I couldn't figure out how to get a dynamic url for the hyperlinks in the markdown using html, so I converted them to markdown. But I can't add a class to a hyperlink in markdown unless it's written in html. It looks like the styling doesn't change when the class is removed, so I'm wondering if we still need a `usa-link` class.

Reply:
Yeah, it looked the same to me but I didn't take the time to compare the classes.  I vote not to worry about it for right now!